addons/website/static/src/scss/website.scss

```diff
@@ -966,10 +966,17 @@ table.table_desc tr td {
     }
 }
 
+.s_list_group {
+    --s_list_group-icon-color: var(--o-cc1-btn-primary);
+}
+
 @for $index from 1 through 5 {
     .o_cc#{$index} .o_dot {
         color: var(--o-cc#{$index}-btn-primary);
     }
+    .o_cc#{$index} .s_list_group {
+        --s_list_group-icon-color: var(--o-cc#{$index}-btn-primary);
+    }
 }
```

Comment:
This is very specific to the `s_list_group` snippet so it should be in the `s_list_group/000.scss` file :+1: If it was a more generic class, it would be okay but not here, just do the same loop in the snippet specific CSS :wink: